Options.lua

```diff
@@ -217,12 +217,14 @@ function WarpDeplete:InitOptions()
             desc = L["Choose how your forces progress will be displayed"],
             sorting = {
               ":percent:",
+              ":pulledpercent:",
```

Comment:
I think I would prefer something like `percentafterpull` here, since we already have `remainingpercentafterpull`. Regretting that I didn't put any word separators or capitalization in the naming here now, but I suppose what's done is done...